src/data/faq_de.json

```
@@ -117,6 +117,62 @@
                     }
                 ]
             },
+            {
+                "title": "Testzertifikat",
+                "id": "test_cert",
+                "accordion": [
+                    {
+                        "title": "Wozu dient das digitales COVID-Testzertifikat?",
+                        "anchor": "test_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Ab der Version 2.4 der Corona-Warn-App (CWA) können Sie ein digitales Testzertifikat über die App anfordern.  Durch die neue Funktion haben Sie die Möglichkeit, ihr negatives Testergebnis durch ein offizielles, digitales COVID-Testzertifikat in Form eines QR-Codes in der CWA nachzuweisen.  Innerhalb der Länder der Europäischen Union kann es als offiziell gültiger Nachweis eines negativen Testergebnisses dienen, den Sie beispielsweise für Reisen nutzen können oder in anderen gesetzlich vorgeschriebenen Fällen, wie z.B. einem Restaurantbesuch. Sie können dann den QR-Code in der CWA vorzeigen."

```

Before innerhalb are 2 whitespaces.

---

"ihr negatives Testergebnis" => "Ihr negatives Testergebnis"
"z.B." => `"z.&nbsp;B."`

---

Double empty spaces 

```suggestion
                            "Ab der Version 2.4 der Corona-Warn-App (CWA) können Sie ein digitales Testzertifikat über die App anfordern. Durch die neue Funktion haben Sie die Möglichkeit, ihr negatives Testergebnis durch ein offizielles, digitales COVID-Testzertifikat in Form eines QR-Codes in der CWA nachzuweisen. Innerhalb der Länder der Europäischen Union kann es als offiziell gültiger Nachweis eines negativen Testergebnisses dienen, den Sie beispielsweise für Reisen nutzen können oder in anderen gesetzlich vorgeschriebenen Fällen, wie z.B. einem Restaurantbesuch. Sie können dann den QR-Code in der CWA vorzeigen."
```

Please note that this suggestion covers the issue found by @DerVogel2020